src/custom/CortiAuth.ts

```diff
@@ -58,6 +67,71 @@ export class Auth extends FernAuth {
         });
     }
 
+    /**
+     * Generate PKCE authorization URL with automatic code verifier generation
+     */
+    public async authorizePkceUrl({
+        clientId,
+        redirectUri,
+    }: AuthorizationCodeClient, options?: Options
+    ): Promise<string> {
+        const codeVerifier = this.generateCodeVerifier();
+
+        if (typeof window !== "undefined" && window.localStorage) {
+            window.localStorage.setItem('code_verifier', codeVerifier);
```

Comment:
Let's name it something more long and specific like "corti_js_sdk_code_verifier" or something similar. Otherwise it is too easy to have some clush with something else.

Plus we should wrap it in try-catch: in some cases (like iframe fx) localStorage is not available and will throw an error. 

Also let's move `code_verifier` to a constant -- to make sure that we use the same one everywhere